SU-SWS/linky_clicky
# Sites 1289

# READY FOR REVIEW

# Summary
- Add Feature for [Views Selective Filters](https://www.drupal.org/project/views_selective_filters)

# Needed By (Date)
- Next week should be fine. I just want to make sure we have test coverage before adding the module to the stack.

# Criticality
- How critical is this PR on a 1-10 scale? 3/10
- It impacts ~8 sites to move from Anchorage to ACSF

# Steps to Test

1. Check out this branch
2. Create a new site on ACSF `dev` `cardinald7` with the `stanford` install profile
3. Run this feature and ensure that it passes
4. Run it again to ensure that the cleanup is sufficient

# Affected Projects or Products
- Anchorage migrations to ACSF

# Associated Issues and/or People
- JIRA ticket: `SITES-1289`
- Other PRs: not yet but there will be one to add the module
- Anyone who should be notified? ( @sherakama @minorwm )

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
